### ISSUE TITLE
Renaming doesn't rename commented method

### ISSUE THREAD

**Issue by artemnikitin** [go-lang-plugin-org/go-lang-idea-plugin#2413]
- Plugin version (or commit hash): 0.11.1203
- IDE name and version: IDEA community 2016.1 (Build #IC-145.258) 
- Java version: java version "1.8.0_20"
- OS name and version: MacOSX 10.11

- What are you trying to do?
Rename method

- What would you expect to happen?
Method name will be renamed for commented method as well

- What happens?
If method usage was commented, then renaming won't rename it

Your issue description goes here.
Please be as detailed as possible
1) Use Refactoring -> Rename on `func example()` to rename method
expected that `//example()` would be renamed as well, but it kept the old name

```go
package main

import "fmt"

func main() {
	//example()
}

func example() {
	fmt.Println("wfwd")
}
```

Or include a screenshot / video of the issue.

**Comment by ignatov** [comment]
Please press Shift+F6 twice and select an appropriate checkbox.

**Comment by artemnikitin** [comment]
Ok, but for Java it works out of box ...

**Comment by artemnikitin** [comment]
For example, if I will edit Java code from the same IDE and do renaming, I will see a window [![screen](https://s3.eu-central-1.amazonaws.com/artemnikitin-blog-pic/idea_go_renaming_comment.png)](https://s3.eu-central-1.amazonaws.com/artemnikitin-blog-pic/idea_go_renaming_comment.png) and then I press `Do Refactor` commented Java code will be edited. But for Go there is no such window while renaming.    
Sorry, but I think that issue should be reopened...

**Comment by bronze1man** [comment]
Is there any way to make Gogland refactor works as this plugin ?
The most difficult function to use in Gogland is that rename part.
If I can not config for it to work,I may always stay in IDEAIC-2017.1.4 and go Lang Plugin 0.171.

The refactor part of Gogland is too complex to rename a lot of method and struct name, you need to  select every reference to that method name to make sure it will not rename comment or strings in other source file.
This plugin can simple rename it, this plugin will find every source code usage of that name which is very good.

**Comment by dlsniper** [comment]
> Is there any way to make Gogland refactor works as this plugin ?

Yes, you can send a PR to it because I'm not going to work on it and there's no other active maintainer for the plugin. I don't want to put any work in it as the JetBrains one does a lot more than this, a lot better.

> Gogland refactor is more stupid and slower than this plugin.

It would be nice to have more than just a rant. 

> Gogland is too complex to rename a lot of method and struct name

I'll check that one out, but as far as I remember, it wasn't working as described at all. Just select the receiver type and rename that.

**Comment by bronze1man** [comment]
> I'll check that one out, but as far as I remember, it wasn't working as described at all. Just select the receiver type and rename that.

If you try rename some name like Error, you will find it slow and stupid.

**Comment by dlsniper** [comment]
> If you try rename some name like Error, you will find it slow and stupid.

Slow and stupid is not something actionable. If you do find something that's an issue, please try and provide steps to replicate the issue

**Comment by bronze1man** [comment]
@dlsniper 
Sorry,I think I just say a rant.
My project is big.The new version may be slow than old one,So I start say a rant.

**Comment by dlsniper** [comment]
@bronze1man I understand it's a problem for you but I cannot debug: it's slow and stupid. Please help me reproduce the issue so that it stops being slow and stupid.

**Comment by zolotov** [comment]
@bronze1man have you tried to invoke Shift+F6 twice and disable searching in string and comments? If yes and it still doesn't work as you want, please file an issue with a particular example to recreate the issue.
